pkgs/development/python-modules/taiga-back/taiga-back-setup.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from setuptools import setup, find_packages
+
+setup(
+    name = 'taiga-back',
```

Comment:
We have no problem to include a setup.py inside the taiga-back repo, but we have to be sure that the requirements are managed correctly (currently we manage the requirements in the requirements.txt file) and we don't want to manage it in two places.